amethyst_utils/src/fps_counter.rs

```diff
@@ -1,9 +1,9 @@
 //! Util Resources
 
+use amethyst_core::{ECSBundle, Result};
 use amethyst_core::timing::{duration_to_nanos, Time};
-use specs::{Fetch, FetchMut, System};
-
 use circular_buffer::CircularBuffer;
+use specs::{DispatcherBuilder, Fetch, FetchMut, System, World};
```

Comment:
Why did you move this import?

Reply:
rustfmt?

Reply:
But it was in the other import group, rustfmt doesn't move betwen import groupts.

Reply:
He ran with normal rustfmt first, not rustfmt-nightly, that might have messed it up.